glTF-BinExporter/RhinoDocGltfConverter.cs

```diff
@@ -0,0 +1,841 @@
+
+                // Accessor Normals
+                var normalsAccessor = new Accessor
+                {
+                    Type = Accessor.TypeEnum.VEC3,
+                    ComponentType = Accessor.ComponentTypeEnum.FLOAT,
+                    Count = dracoGeoInfo.normalsNum,
+                    Min = dracoGeoInfo.normalsMin,
+                    Max = dracoGeoInfo.normalsMax,
+                    ByteOffset = 0,
+                };
+
+                int normalsAccessorIdx = dummy.Accessors.AddAndReturnIndex(normalsAccessor);
+
+                // Accessor TexCoords
+                var texCoordsAccessor = new Accessor
+                {
+                    Type = Accessor.TypeEnum.VEC2,
+                    ComponentType = Accessor.ComponentTypeEnum.FLOAT,
+                    Count = dracoGeoInfo.texCoordsNum,
+                    Min = dracoGeoInfo.texCoordsMin,
+                    Max = dracoGeoInfo.texCoordsMax,
+                    ByteOffset = 0,
+                };
+                
+                int texCoordsAccessorIdx = dummy.Accessors.AddAndReturnIndex(texCoordsAccessor);
+
+                var primitive = new MeshPrimitive()
+                {
+                    Attributes = new Dictionary<string, int>()
+                    {
+                        { Constants.PositionAttributeTag, vtxAccessorIdx },
+                        { Constants.NormalAttributeTag, normalsAccessorIdx },
+                        { Constants.TexCoord0AttributeTag, texCoordsAccessorIdx },
+                    },
+                    Indices = idsAccessorIdx,
+                    Material = materialIndex,
+                    Extensions = new Dictionary<string, object>()
+                    {
+                        {
+                            Constants.DracoMeshCompressionExtensionTag,
+                            new
+                            {
+                                bufferView = compMeshBufferViewIdx,
+                                attributes = new
+                                {
+                                    POSITION = 0,
+                                    NORMAL = 1,
+                                    TEXCOORD_0 = 2
+                                }
+                            }
+                        }
+                    }
+                };
+
+                // Create mesh
+                primitives.Add(primitive);
+            }
+            var mesh = new glTFLoader.Schema.Mesh()
+            {
+                Primitives = primitives.ToArray(),
+            };
+            
+            int meshIndex = dummy.Meshes.AddAndReturnIndex(mesh);
+            
+            var node = new Node()
+            {
+                Mesh = meshIndex,
+            };
+            int nodeIndex = dummy.Nodes.AddAndReturnIndex(node);
+
+            dummy.Scenes[dummy.Scene].Nodes.Add(nodeIndex);
+        }
+
+        public DracoGeometryInfo AddDracoGeometry(DracoCompression dracoCompression)
```

Comment:
I'm tempted to move the object conversion entirely to a new file instead of having it shoehorned in with the entirety of the document conversion. It'd make more sense there and static as well.

Reply:
Logged as #29